### PR TITLE
feat: getInternalAddress method

### PR DESCRIPTION
**Pull request by sergdeus**
Added getInternalAddress method, that uses Rust method

```
//Return a derived address using the internal descriptor,
pub fn get_internal_address(
    wallet: RustOpaque<WalletInstance>,
    address_index: AddressIndex,
) -> anyhow::Result<AddressInfo, anyhow::Error> {
    match wallet.get_internal_address(address_index) {
        Ok(e) => Ok(e),
        Err(e) => anyhow::bail!("{:?}", e),
    }
}
```

### CODE REVIEW

File: lib/src/generated/bridge_definitions.dart [hunk 1]
```diff
@@ -97,16 +86,12 @@ abstract class Rust {
 
   FlutterRustBridgeTaskConstMeta get kBumpFeeTxBuilderFinishConstMeta;
 
-  Future<BdkDescriptor> newDescriptor(
```
Comment (BitcoinZavior) [inline lib/src/generated/bridge_definitions.dart:100]: Would it be possible not to club style changes in the same PR?
It makes checking for changes a little difficult.

**Comment by BitcoinZavior** [comment]
@sergdeus Thanks for the PR, this will be a nice addition!

**Comment by BitcoinZavior** [comment]
Other than the style being changed I don't really have any other comments.
Can we maintain the same style, please?

**Comment by sergdeus** [comment]
> Other than the style being changed I don't really have any other comments. Can we maintain the same style, please?

Yep, sorry for the styles.
I've applied `flutter format .` command. Is it ok?
In this way, it is possible to keep the same styles despite on IDE lint setting.

**Comment by BitcoinZavior** [comment]
> > Other than the style being changed I don't really have any other comments. Can we maintain the same style, please?
> 
> Yep, sorry for the styles. I've applied `flutter format .` command. Is it ok? In this way, it is possible to keep the same styles despite on IDE lint setting.

Perfect, thanks!

**Review by BitcoinZavior — APPROVED** [review]

**Pull request merged**